types/koa/index.d.ts

```diff
@@ -551,7 +551,7 @@ declare namespace Application {
         /**
          * Custom properties.
          */
-        [key: string]: any;
+        [key: keyof any]: any;
```

Comment:
Why not `string | symbol`?

Reply:
Because `keyof any` results in `string | number | symbol`, i.e. [`PropertyKey`](https://github.com/microsoft/TypeScript/blob/v5.0.4/src/lib/es5.d.ts#L90)

Reply:
Then you should use `PropertyKey` type or primitive types... You should also add some tests for it.